lib/service/storage/elasticsearch.ts

```diff
@@ -738,7 +811,15 @@ class ElasticSearch extends Service {
     collection,
     id,
     content,
-    { refresh, userId = null, injectKuzzleMeta = true } = {},
+    {
+      refresh,
+      userId = null,
+      injectKuzzleMeta = true,
+    }: {
+      refresh?: boolean | "wait_for";
+      userId?: string;
+      injectKuzzleMeta?: boolean;
+    } = {},
```

Comment:
```suggestion
    {
      refresh,
      userId = null,
      injectKuzzleMeta = true,
    }: Exclude<KRequestParams, "retryOnConflict" | "timeout"> = {},
```

With new `KRequestParams` type we can compose type like it, to DRY